src/patternfly/components/Content/content.scss

```diff
@@ -15,39 +15,22 @@
   --pf-c-content--h1--FontWeight: var(--pf-global--FontWeight--normal);
 
   // h2 - Secondary title
-  --pf-c-content--h2--MarginTop: var(--pf-global--spacer--lg);
-  --pf-c-content--h2--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-content--h2--LineHeight: var(--pf-global--LineHeight--sm);
   --pf-c-content--h2--FontSize: var(--pf-global--FontSize--2xl);
-  --pf-c-content--h2--FontWeight: var(--pf-global--FontWeight--normal);
 
   // h3 - Sub-title
-  --pf-c-content--h3--MarginTop: var(--pf-global--spacer--lg);
-  --pf-c-content--h3--MarginBottom: var(--pf-global--spacer--md);
   --pf-c-content--h3--LineHeight: var(--pf-global--LineHeight--md);
   --pf-c-content--h3--FontSize: var(--pf-global--FontSize--xl);
-  --pf-c-content--h3--FontWeight: var(--pf-global--FontWeight--normal);
 
   // h4
-  --pf-c-content--h4--MarginTop: var(--pf-global--spacer--lg);
-  --pf-c-content--h4--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-content--h4--LineHeight: var(--pf-global--LineHeight--md);
   --pf-c-content--h4--FontSize: var(--pf-global--FontSize--lg);
   --pf-c-content--h4--FontWeight: var(--pf-global--FontWeight--semi-bold);
 
   // h5
-  --pf-c-content--h5--MarginTop: var(--pf-global--spacer--lg);
-  --pf-c-content--h5--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-content--h5--LineHeight: var(--pf-global--LineHeight--md);
   --pf-c-content--h5--FontSize: var(--pf-global--FontSize--md);
-  --pf-c-content--h5--FontWeight: var(--pf-global--FontWeight--semi-bold);
 
   // h6
-  --pf-c-content--h6--MarginTop: var(--pf-global--spacer--lg);
   --pf-c-content--h6--MarginBottom: var(--pf-global--spacer--xs);
-  --pf-c-content--h6--LineHeight: var(--pf-global--LineHeight--md);
   --pf-c-content--h6--FontSize: var(--pf-global--FontSize--md);
-  --pf-c-content--h6--FontWeight: var(--pf-global--FontWeight--semi-bold);
 
   // Small text
   --pf-c-content--small--MarginBottom: var(--pf-global--spacer--md);
```

Comment:
could you remove this and use: `--pf-c-content--MarginBottom: var(--pf-global--spacer--md);` since they share the same value.

Reply:
@mcoker if not (since we're not certain on this yet) let me know and I'll approve the issue. Everything else looks really good.

Reply:
oh good catch! `margin-bottom` was already defined on `small` :) removed it.

https://github.com/patternfly/patternfly-next/blob/947cfbde794ff595f7adb14934ffc22fb8a9b79d/src/patternfly/components/Content/content.scss#L88-L92